core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.spec.ts

```diff
@@ -246,8 +250,8 @@ describe('Contributor dashboard Admin page', () => {
     expect(component.selectedLanguage.language).toBe('English');
     expect(component.selectedLanguage.id).toBe('en');
 
-    component.selectLanguage('العربية (Arabic)');
-    expect(component.selectedLanguage.language).toBe('العربية (Arabic)');
+    component.selectLanguage('Arabic (العربية)');
+    expect(component.selectedLanguage.language).toBe('Arabic (العربية)');
```

Comment:
Can you also do the same thing here as you did in lines 173-180 within this file?

Reply:
Done.